examples/python/evm/account.sign_typed_data.py

```diff
@@ -0,0 +1,38 @@
+# Usage: uv run python evm/sign_typed_data.py
+
+import asyncio
+
+from cdp import CdpClient
+from cdp.openapi_client.models.eip712_domain import EIP712Domain
+from cdp.openapi_client.models.eip712_message import EIP712Message
```

Comment:
We want to abstract away the openapi_client from SDK users. I suggest re-exporting these from the top-level `cdp` module